packages/cli/src/config/extension.ts

```diff
@@ -686,15 +704,14 @@ export function loadExtensionConfig(
   }
   try {
     const configContent = fs.readFileSync(configFilePath, 'utf-8');
-    const rawConfig = JSON.parse(configContent) as ExtensionConfig;
-    if (!rawConfig.name || !rawConfig.version) {
-      throw new Error(
-        `Invalid configuration in ${configFilePath}: missing ${!rawConfig.name ? '"name"' : '"version"'}`,
-      );
-    }
-    const installDir = new ExtensionStorage(rawConfig.name).getExtensionDir();
+    const rawConfig = JSON.parse(configContent);
```

Comment:
nit JSON.parse() returns an `any`.  I'd feel a bit better if we cast this to `as unknown`